pkg/scheduler/schedule_one.go

```diff
@@ -129,6 +129,15 @@ func (sched *Scheduler) ScheduleOne(ctx context.Context) {
 			sched.handleBindingCycleError(bindingCycleCtx, state, fwk, assumedPodInfo, start, scheduleResult, status)
 			return
 		}
+		// Update the PodCondition condition to reflect that the pod is scheduled.
+		podutil.UpdatePodCondition(&pod.Status, &v1.PodCondition{
+			Type:   v1.PodScheduled,
+			Status: v1.ConditionTrue,
+		})
+
+		// Usually, DonePod is called inside the scheduling queue,
+		// but in this case, we need to call it here because this Pod won't go back to the scheduling queue.
+		sched.SchedulingQueue.Done(assumedPodInfo.Pod.UID)
```

Comment:
Is this change relevant to the issue?
I thought `Done` should already be called inside the binding cycle if pod passes `permit` and `prebind`, no?